server/routes/bookings/bookingDetailsController.ts

```diff
@@ -0,0 +1,48 @@
+import type { RequestHandler } from 'express'
+import { ValidationChain, param, validationResult } from 'express-validator'
+import { VisitService } from '../../services'
+import getPrisonInformation from '../../constants/prisonInformation'
+import paths from '../../constants/paths'
+
+export default class BookingDetailsController {
+  public constructor(private readonly visitService: VisitService) {}
+
+  public view(): RequestHandler {
+    return async (req, res) => {
+      const { booker, bookings } = req.session
+
+      const errors = validationResult(req)
+      if (!errors.isEmpty()) {
+        req.flash('errors', errors.array())
```

Comment:
```suggestion
```
Don't want to set flash here as we don't load it where you're redirecting to. This means it could randomly appear on the next page they hit where `flash('errors')` is read!